eng/pipelines/templates/jobs/ci.yml

```
@@ -233,10 +235,22 @@ jobs:
       MatrixFilters: ${{ parameters.MatrixFilters }}
       MatrixReplace: ${{ parameters.MatrixReplace }}
       CloudConfig:
-        Cloud: Public
+        Cloud: public
+      ${{ if ne(parameters.TestDependsOnDependency, 'not-specified') }}:
+        ProjectListOverrideFileProperty: 'ProjectListOverrideFileProperty'
```

```suggestion
        ProjectListOverrideFilePropertyName: 'ProjectListOverrideFile'
```

I wouldn't expect us to have the Property prefix on the actually name itself only on the parameters that are talking about the name.